### PR TITLE
Populate Home Page with grid of RequestCards

### PR DESCRIPTION
**Pull request by pforderique**
Added filterable request grid view of user's own requests

### CODE REVIEW

File: src/components/RequestCard.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          {{ props.request.upvotingUsers.length }}
+          <a href="#" class="btn btn-sm btn-primary">▲</a>
+        </span>
+      </h5>
+      <h6 class="card-subtitle mb-2 text-muted">{{ props.request.space }}</h6>
+      <p class="card-text">
+        {{ props.request.textContent }}
+      </p>
+      <p class="emphasized">Created {{ props.request.dateCreated }}</p>
+      <div
```
Comment (camtheman256) [inline src/components/RequestCard.vue]: For status here, I might recommend a computed property for `status` (https://vuejs.org/guide/essentials/computed.html) with a `:class` property to turn on and off the different bg's: https://vuejs.org/guide/essentials/class-and-style.html, so then you could get rid of the v-if and condense these all into one div

File: src/components/RequestsGrid.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+<template>
+  <div>
+    <section class="spanned">
+      <h5>My Requests</h5>
+      <RequestSearch @filter="updateFilter($event)" />
+    </section>
+    <div
+      class="row bottom-buffer"
+      v-for="index in Math.floor(
```
Comment (camtheman256) [inline src/components/RequestsGrid.vue]: I might try a regular `v-for` here and then use the `col-sm-4` to reflow the extra cards here automatically, so you should be able to get away with just one `<RequestCard :request="request" />` component, something like this: https://getbootstrap.com/docs/5.2/layout/grid/#row-columns

File: src/components/RequestSearch.vue [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      class="form-control"
+      ref="searchbox"
+      placeholder="filter requests"
+    />
+
+    <div class="input-group-append">
+      <button
+        class="btn btn-outline-secondary"
+        type="button"
+        @click="$emit('filter', searchbox.value)"
```
Comment (camtheman256) [inline src/components/RequestSearch.vue]: You could bind this to `@input` if you wanted to make it immediately reactive and nix the search button, but up to you

Comment (pforderique) [reply src/components/RequestSearch.vue]: I like this better than having to click enter each time!

**Pull request merged**